evil-collection-evil-search.el

```diff
@@ -31,6 +31,8 @@
 ;;; Code:
 (require 'evil-search)
 
+(defvar evil-collection-evil-search-enabled (eq evil-search-module 'evil-search))
```

Comment:
Hmm, I think this should rather be a function to make it dynamic.  Otherwise, if evil-search-module changes it won't be reflected after definition of `evil-collection-evil-search-enabled`.